pmm-app/src/pmm-dbaas/components/Kubernetes/KubernetesInventory.test.tsx

```diff
@@ -20,7 +20,7 @@ describe('KubernetesInventory::', () => {
 
     expect(rows.length).toBe(kubernetesStub.length + 1);
   });
-  it('cancels delete cluster', () => {
+  xit('cancels delete cluster', () => {
```

Comment:
This should still be working.

Reply:
Ooops, forgot to add real tests after copy-paste, added